### PR TITLE
circular_queue using Java

### PR DESCRIPTION
**Pull request by Rishikeshrajrxl**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#1741 
(Write your answer here.)

### CODE REVIEW

**Comment by Rishikeshrajrxl** [comment]
@shraddhavp, please review

File: Java/ds/circular_queue.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+                    case 5:
+                        {
+                            System.exit(0);
+                        }
+                        break;
+                    default:
+                        System.out.println("Invalid Choice ! Try again.");
+                        break;
+                }
+        }while(true);
```
Comment (HarshCasper) [inline Java/ds/circular_queue.java:131]: Indent the piece of Code

File: Java/ds/circular_queue.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        }
+        System.out.print("Deleted : "+ data);  
+    }
+    //////////////////////////////////////////////////////////////
+    public static void main(String []args)  //main function
+    {
+        Scanner sc=new Scanner(System.in);
+        System.out.print("Enter the size of queue : ");
+        int size = sc.nextInt();
+        Main m = new Main(size);
```
Comment (HarshCasper) [inline Java/ds/circular_queue.java:76]: Follow a consistent indentation

File: Java/ds/circular_queue.java [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+        else if(front ==size-1)
+        {
+            front =0;
+        }
+        else{
+            front++;
+        }
+        System.out.print("Deleted : "+ data);  
+    }
+    //////////////////////////////////////////////////////////////
```
Comment (HarshCasper) [inline Java/ds/circular_queue.java:70]: Remove such comments from the entire Code

File: Java/ds/circular_queue.java [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+    public static void Dequeue()
+    {
+        int data=arr[front];
+        arr[front]=-1;
+        if(front ==rear)
+        {
+            front=-1; rear=-1;
+        }
+        else if(front ==size-1)
```
Comment (HarshCasper) [inline Java/ds/circular_queue.java:61]: Inconsistent Indentation.

File: Java/ds/circular_queue.java [hunk 5]
```diff
@@ -0,0 +1,133 @@
+import java.util.*;
+import java.lang.*;
+
+public class Main
+{
+    public  static int front;
+    public static int rear;
+    public static int arr[];
+    public static int size;
+    Main(int n) //Constructor
```
Comment (HarshCasper) [inline Java/ds/circular_queue.java:10]: Use Comments above the LOC not beside them. Also don't put simplistic comments.

File: Java/ds/circular_queue.java [hunk 6]
```diff
@@ -0,0 +1,133 @@
+import java.util.*;
```
Comment (HarshCasper) [inline Java/ds/circular_queue.java:1]: Add a beginner pitch about the Code that you have written

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Make README changes as well

**Comment by HarshCasper** [comment]
Closing the PR cause of no updates. Let me know if you would like this to be reopened.

**Comment by Rishikeshrajrxl** [comment]
> Closing the PR cause of no updates. Let me know if you would like this to be reopened.

@HarshCasper , please reopen this PR, I will do the changes. And sorry for delay.

**Pull request closed without merging**